core-common/src/main/java/org/zalando/nakadi/domain/NakadiRecordMetadata.java

```diff
@@ -2,35 +2,36 @@
 
 public class NakadiRecordMetadata {
```

Comment:
The `NakadiRecordMetadata` name is very confusing. 
A better name should indicate that it is a response or status of trying to publish an event. 
Something like `EventPublishResult`.